### PR TITLE
Added support in ONC ingestion for updated keywords

### PR DESCRIPTION
**Pull request by twilson271828**
The Hayabusa 2 team changed/removed some keyword values in the fit header file.  This PR modifies
hsb2onc2isis to handle this.  It will also be backwards compatible with fit files using the old format.

## Description

- Uploaded new translation files for BandBin/Instrument/Kernel groups.  These were named differently than the original translation files to ensure backwards compatibility.

- Modified hsb2onc2isis to detect whether it was looking at an older fits file or one with updated keywords.  To do this, I check for the existence of P_ID in the fits label.  This was renamed to NAIFID, but I think it's a safe bet that any valid older version fits file is going to have this keyword.

- Also added a check to determine if geometric correction has been applied, and added the DistortionCorrection keyword to the Instrument group in the output cube label.  If geometric corrections have been applied, this value is 1.  If not, it is 0.

- The file filter for the input file in the XML of hyb2onc2isis was set to *.lbl/*.LBL.  I don't know why, but I changed this to *.fit.

- Changed the data location in IsisPreferences to point to a local data area and not the $ISIS3DATA area (it points to:  /usgs/cpkgs/isis3/datalocal/hayabusa2/).  

## Motivation and Context

The Hayabusa 2 Team wants it in ISIS3.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by twilson271828** [comment]
Still need to add testing to this PR.

File: isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml [hunk 1]
```diff
@@ -53,8 +54,8 @@
        cameras (ONC-W1,ONC-W2). 
     </p>
     <p>
-       The wide-angle cameras each have a square 65.24 x 65.24 deg. field of view (FOV). The wide-angle 
-       camera ONC-W1 and ONC-W2 have focal lengths of
+       The wide-angle cameras each have a square 65.24 x 65.24 deg. field of view (FOV).
+       The wide-angle camera ONC-W1 and ONC-W2 have focal lengths of
```
Comment (scsides) [inline isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml]: Suggest removing details about the instrument: Including values like the focal length here makes it easy to have incorrect info here. The instrument addendum kernel is the definitive source for these values. Most people who update a kernel file will not look in the app docs to make sure they agree.

File: isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml [hunk 2]
```diff
@@ -53,8 +54,8 @@
        cameras (ONC-W1,ONC-W2).
```
Comment (scsides) [inline isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml]: The telescopic camera is also supported

File: isis/src/hayabusa2/apps/hyb2onc2isis/main.cpp [hunk 3]
```diff
@@ -70,21 +103,56 @@ void IsisMain () {
...
+  catch(IException &e) {
+    QString msg = "Unable to read TARGET from input file ["
+        +FileName(ui.GetFileName("FROM")).expanded() + "]";
+    throw IException(e, IException::Io,msg, _FILEINFO_);
+  }
+  if (target=="SKY") {
+    instGrp["TargetName"] = "RYUGU";
+  }
+
+  if (distortionCorrection) {
```
Comment (scsides) [inline isis/src/hayabusa2/apps/hyb2onc2isis/main.cpp:132]: Suggest using Yes or No for the values

File: isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml (hunk 1 above)
Comment (twilson271828) [reply isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml]: I'm going to remove the section about the specification for the ONC-T filters as well, then.

File: isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml (hunk 2 above)
Comment (twilson271828) [reply isis/src/hayabusa2/apps/hyb2onc2isis/hyb2onc2isis.xml]: Reference to ONC-T is made on the line above this.

File: isis/src/hayabusa2/apps/hyb2onc2isis/main.cpp (hunk 3 above)
Comment (twilson271828) [reply isis/src/hayabusa2/apps/hyb2onc2isis/main.cpp:132]: Fixed it.

**Pull request merged**